python/tvm/autotvm/measure/measure.py

```diff
@@ -97,7 +112,17 @@ def set_task(self, task, build_kwargs=None):
             The additional kwargs for build function
         """
         self.task = task
-        self.build_kwargs = build_kwargs
+        self.build_kwargs = build_kwargs if build_kwargs is not None else {}
```

Comment:
can you create a copy of build_kwargs:
`self.build_kwargs = dict(build_kwargs.items()) if build_kwargs is not None else {}`